android/src/main/java/com/reactnativekalturaplayer/KalturaPlayerRN.kt

```diff
@@ -849,16 +846,15 @@ class KalturaPlayerRN(
      * This method checks if the Player type is OVP/OTT
      * but still app wants to play a media using URL or DRM license URL
      * instead of using our backend.
+     * @param assetURL Playback URL for basic player
      *
      * @return `true` if basic playback required
      */
-    private fun isBasicPlaybackRequired(basicMediaAsset: String?): Boolean {
-        if ((basicMediaAsset != null) && (playerType != null) && (playerType != KalturaPlayer.Type.basic)) {
-            val mediaAsset = getParsedJson(
-                    basicMediaAsset,
-                    BasicMediaAsset::class.java
-            )
-            return mediaAsset != null && mediaAsset.mediaFormat != null
+    private fun isBasicPlaybackRequired(assetURL: String?): Boolean {
+        if (!TextUtils.isEmpty(assetURL) && playerType != null && playerType != KalturaPlayer.Type.basic) {
+            val isBasicPlayback = URLUtil.isValidUrl(assetURL) && PKMediaFormat.valueOfUrl(assetURL) != null
```

Comment:
PKMediaFormat.valueOfUrl(assetURL), What if there is no media extension in URL and there is valid mediaAsset having media format. Need to raise a ticke to fix such scenario.

Reply:
FEC-12714